Bottom: 8ba682a0c3c33065ac98b25eb7d89e6bd3692e6a
Top:    c4c9513c12a6d487875d8f4d2a1837ce06d96488
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:57:31 +0000

hid/batch: Add some missing header files for prototypes


---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index 8d53d8c..9715c09 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -8,10 +8,14 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "global.h"
 #include "hid.h"
 #include "data.h"
+#include "misc.h"
+#include "hid.h"
+#include "../hidint.h"
 
 #include "hid/common/actions.h"
 #include "hid/common/hidinit.h"
